data/migrations/20200124081059_my-projects.js

```diff
@@ -0,0 +1,61 @@
+exports.up = function(knex) {
+  return knex.schema
+    .createTable("projects", table => {
+      table.increments("projectID");
+      table
+        .string("title", 128)
+        .notNullable()
+        .unique();
+      table.text("projectDescription");
+      table.boolean("completed").defaultTo(false);
+    })
+    .createTable("resources", table => {
+      table.increments("resourceID");
+      table
+        .string("resourceName", 140)
+        .notNullable()
+        .unique();
+      table.text("resourceDescription");
```

Comment:
Appending the table's name to the attributes' names is not necessary. E.g resouceName, should be `name`